### ISSUE TITLE
Unable to get Email-handler to work, umb-8 / formulate 3

### ISSUE THREAD

**Issue by kristoffer-tvera** [rhythmagency/formulate#161]
Umbraco version 8.2.1
Installed Formulate through Nuget
`<package id="Formulate" version="3.0.0" targetFramework="net472" />`
`<package id="Formulate.Binaries" version="3.0.0" targetFramework="net472" />`

I've set up my SMTP block correctly (can verify this through sending a simple email in a surface controller with nonconfigured `new SmtpClient()` and a "Hello World" MailMessage )

I've doublechecked the fields, and tried multiple different combinations.

Nothing is logged to error / serilog.

The forms are successfully logged to database / backend, and I can inspect them there, but I am unable to get the email-handler to do anything at all.

**Comment by kristoffer-tvera** [comment]
Ok, so I resolved this issue after combing through your source-code.

First clue that I missed, was on this site:
http://www.formulate.rocks/installing

`The first time the website is run with Formulate installed, Formulate will modify some files (e.g., it will change the web.config to point to the Formulate configuration files).`

I didnt experience that the web.config was changed at all, but the clue here was the Formulate configuration files, which doesnt seem to be referenced anywhere else.

This is emphasized after reading through the code that sends emails through the default email handler; and finding references to a whitelist
https://github.com/rhythmagency/formulate/blob/v3/master/src/formulate.app/Forms/Handlers/Email/EmailHandler.cs#L503

So after finding this, I found that there is indeed a config file here:
https://github.com/rhythmagency/formulate/blob/v3/master/src/formulate.app/App_Plugins/formulate/FormulateConfiguration.json

With whitelisting enabled by default, and the whitelist that is preconfigured only lets through emails from `person@domain.com`.

Is this configuration available anywhere in the backoffice, or somewhere more obvious? Figuring this out earlier would have saved me a couple of hours :S
